src/engine.rs

```diff
@@ -1250,13 +1255,14 @@ impl<'a, P: ProcessRun> PubPoint<'a, P> {
             self.cert.cert(), self.run.validation.strict
         ) {
             Ok(cert) => cert,
-            Err(_) => {
-                warn!("{}: CA certificate failed to validate.", uri);
+            Err(err) => {
+                warn!("{}: {}.", uri, err);
```

Comment:
With both CA certs and router certs having file extension `.cer`, are we still able to determine which of the two types the error message describes without depending on the actual (contents of) `err`?

Would it make sense to add back 'CA' and 'router' in the `warn!`?

Reply:
I believe the error messages contain the type of certificate if it is relevant. I think not including the type when it doesn’t matter might be clearer?